src/main/webapp/main-page.html

```diff
@@ -44,9 +45,12 @@ <h1> Filters</h1>
 
           <div id="table-filters"></div>
 
-          <input type="submit" id="submit" value="submit" onClick=" return isFilterInputEmpty()">
+          <input type="submit" id="submit" value="submit" onClick="return isFilterInputEmpty()">
         </form>  
       </div>
     <div id="user"></div>
+    <div id="tables"></div>
+
+    <div id="sql"></div>
```

Comment:
So this is the div that displays what the user selected ?

Reply:
yes